x-pack/plugins/security_solution/public/common/components/user_privileges/index.ts

```diff
@@ -0,0 +1,13 @@
+/*
```

Comment:
split out the Context code from the Hook so that we can better mock the hook in jest